snooty.toml

```
@@ -10,6 +10,15 @@ package-branch = "testing"
 package-version = "4.4.0"
 package-name-org = "mongodb-org"
 version = "4.0"
+rst-title = "ReStructuredText"
```

I think this should always start lower-case even when you're using Title case. The Primer is only piece of RST documentation that uses a capital "R" in it's titling and this seems to be an unintentional exception. See the [RST documentation list](https://docutils.sourceforge.io/rst.html#user-documentation).

```suggestion
rst-title = "reStructuredText"
```

---

I wasn't sure what's correct since RST, reST, and ReST are all valid abbreviations which would seem to imply that capitalization is ok. I have no issues with representing it with lowercase though.